index.html

```diff
@@ -49,5 +49,203 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="/image/Tag.png" alt="html" /></li>
+          <li><img src="/image/Tag1.png" alt="css" /></li>
+          <li><img src="/image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+
+      <!-- fourth work section -->
+
+      <div class="card">
+        <img class="img" src="/image/Snapshoot4.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="/image/counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="/image/Tag.png" alt="html" /></li>
+          <li><img src="/image/Tag1.png" alt="css" /></li>
+          <li><img src="/image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+<div class="bg-white">
+    <section id="aboutme">
+      <div class="me">
+        <h3 class="aboutme">About Myself</h3>
+        <p class="p">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <h4 class="h4-connect">LET'S CONNECT</h4>
+
+        <ul class="flex item-center">
+          <li class="icon-tag">
+            <a href="#"><img src="/image/twitter.png" alt="twitter" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/linkedin.png" alt="linkedin" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/m.png" alt="m" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/git.png" alt="Github" /></a>
+          </li>
+          <li class="icon-tag">
+            <a href="#"><img src="/image/angelist.png" alt="angelist" /></a>
+          </li>
+        </ul>
+        <button>Get my resume</button>
+      </div>
+      <div class="base-about-myself">
+      <h4>Languages</h4>
+      <ul>
+        <li><img src="/image/framejs.png" alt="frameJs" /></li>
+        <li><img src="/image/framehtml.png" alt="framehtml" /></li>
+        <li><img src="/image/framecss.png" alt="framecss" /></li>
+      </ul>
+      <div>
+        <ul>
+          <li class="frameworks"><p>Frameworks ></p></li> <hr>
+          <li class="skills"><p>Skills ></p></li>
+          <hr>
+        </ul>
+       
+      </div>
+    </div>
+    </section>
```

Comment:
- Kindly ensure that the color, font family, font size, line height, margins (top, bottom, left, right) and font weight of your about section i is stick to the Figma design you chose as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). You can check out the Figma design specification for the exact properties used👍🏽
 
- I suggest importing this font family I show you below  so that you can apply it in your all  body :+1: :grinning: :grinning: :grinning: 

```
@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");

 font-family: "Poppins", sans-serif;

```


## your design :-1: 

![image](https://user-images.githubusercontent.com/95297251/192108756-e4e6d15e-21c4-4b0e-a5c2-816fd9fdaaeb.png)


## Figma design :+1: 

![image](https://user-images.githubusercontent.com/95297251/192108785-82627f0e-b089-434f-8f5c-f2257d6c557a.png)